x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/aggregation/X-ValuesAggregator.java.st

```diff
@@ -272,59 +273,123 @@ $endif$
             if (values.size() == 0) {
```

Comment:
I think it's worth adding javadoc to this to explain what it's goals are. It's long enough that it really needs it.

Reply:
pushed a patch for this.